scene/resources/animation.cpp

```diff
@@ -427,6 +427,15 @@ bool Animation::_set(const StringName &p_name, const Variant &p_value) {
 		} else {
 			return false;
 		}
+#ifndef DISABLE_DEPRECATED
+	} else if (prop_name == "loop"){
+		if (p_value) {
```

Comment:
```suggestion
		bool loop = p_value;
		if (loop) {
```

To ensure its an actual bool value